test/dummy/app/controllers/overrides/sessions_controller.rb

```diff
@@ -5,7 +5,7 @@ class SessionsController < DeviseTokenAuth::SessionsController
     def create
       @resource = resource_class.find_by_email(resource_params[:email])
 
-      if @resource and valid_params?(:email, resource_params[:email]) and @resource.valid_password?(resource_params[:password]) and @resource.confirmed?
+      if @resource and resource_params[:password].present? and @resource.valid_password?(resource_params[:password]) and @resource.confirmed?
```

Comment:
If `valid_params?` wasn't changed in `sessions_controller`, this wouldn't be needed